corehq/apps/receiverwrapper/tests/test_submit_errors.py

```diff
@@ -28,11 +29,16 @@
     use_sql_backend,
 )
 from corehq.middleware import OPENROSA_VERSION_HEADER
-from corehq.util.test_utils import TestFileMixin, flag_enabled
+from corehq.util.test_utils import TestFileMixin, flag_enabled, capture_log_output
 
 FORM_WITH_CASE_ID = 'ad38211be256653bceac8e2156475666'
 
 
+def tmpfile(mode='w', *args, **kwargs):
```

Comment:
moved this here so I can delete other code: https://github.com/dimagi/commcare-hq/pull/25581